pkgs/servers/web-apps/discourse/plugins/discourse-calendar/Gemfile

```diff
@@ -2,4 +2,7 @@
 
 source "https://rubygems.org"
 
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
```

Comment:
@talyz Looks like maybe it is unnecessary with bundler 2: https://github.com/rubygems/bundler/issues/5476 Is there some other reason it is needed?

Reply:
It was added by bundler, so I don't know :shrug: In our case, it's probably fine to remove it, since we're fetching the dependencies ourselves anyway, but it will most likely just be added back the next time `update.py` is run.